qtconsole/console_widget.py

```diff
@@ -898,7 +898,7 @@ def _tab_pressed(self):
         """ Called when the tab key is pressed. Returns whether to continue
             processing the event.
         """
-        return False
+        return True
```

Comment:
Not sure what the larger implications are of this change, but without it, the event handling code for tab is untestable, because it is guarded by [an if](https://github.com/JeremySikes73/qtconsole/blob/2a903ef6988a2da8f5cb09cb881915e9effd245e/qtconsole/console_widget.py#L1337)

Reply:
@wmvanvliet , no problem here?